modules/atomic-widgets/prop-types/primitives/supports-shorthanded-value.php

```diff
@@ -17,14 +17,21 @@ public function validate( $value ): bool {
 			return ! $this->is_required();
 		}
 
-		if ( $this->validate_value( $value ) ) {
-			return true;
+		if ( ! $this->is_transformable( $value ) ) {
+			return $this->validate_value( $value );
```

Comment:
This class should be remove

Reply:
We'll take it in a designated PR as it requires some changes to our test-atomic-widget-base tests, and there's some consultation to have with the team regarding how to target use-cases like Size_Prop_Type (where both `unit` and `size` keys should also be changed to the non-shorthanded string/number values - if we choose to go straightforward with this approach)